src/test/java/net/sf/jabref/importer/fileformat/RISImporterTest.java

```diff
@@ -28,82 +26,196 @@ public void setUp() throws Exception {
     }
 
     @Test
+    public void testGetFormatName() {
+        RisImporter risImporter = new RisImporter();
+        Assert.assertEquals(risImporter.getFormatName(), "RIS");
+    }
+
+    @Test
+    public void testGetCLIId() {
+        RisImporter risImporter = new RisImporter();
+        Assert.assertEquals(risImporter.getCLIId(), "ris");
+    }
+
+    @Test
     public void testIsRecognizedFormat() throws IOException {
 
         RisImporter importer = new RisImporter();
-        try (InputStream stream = RISImporterTest.class
-                .getResourceAsStream("RisImporterTest1.ris")) {
+
+        try (InputStream stream = RISImporterTest.class.getResourceAsStream("RisImporterTest1.ris")) {
             Assert.assertTrue(importer.isRecognizedFormat(stream));
         }
+        try (InputStream stream = RISImporterTest.class.getResourceAsStream("RisImporterTest2.ris")) {
+            Assert.assertFalse(importer.isRecognizedFormat(stream));
+        }
     }
 
+
     @Test
-    public void testProcessSubSup() {
+    public void testImportEntries() throws IOException {
+        RisImporter importer = new RisImporter();
 
-        HashMap<String, String> hm = new HashMap<>();
-        hm.put("title", "/sub 3/");
-        IsiImporter.processSubSup(hm);
-        Assert.assertEquals("$_3$", hm.get("title"));
+        try (InputStream stream = RISImporterTest.class.getResourceAsStream("RisImporterTest1.ris")) {
+            List<BibEntry> entries = importer.importEntries(stream, new OutputPrinterToNull());
+            BibEntry entry = entries.get(0);
+
+            Assert.assertEquals(1, entries.size());
```

Comment:
Create a new `expected` bibentry and compare the parsed list against it using the methods of BibtexEntryAssert (see the recent discussion in the other importer PR)